### PR TITLE
Added donation buttons & links.

### LINKED ISSUE #332: Add donation link
### Problem

Current README doesn't include any mention on how to support the project.

### Solution

Add links to https://okturtles.com/donate/ and https://groupincome.org/donate/ to README using buttons and whatnot.

### PR DESCRIPTION
**Pull request by dotmacro**
Closes #332

### CODE REVIEW

**Review by sandrina-p — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![Gitter](https://badges.gitter.im/okTurtles/group-income.svg)](https://gitter.im/okTurtles/group-income) [![Build Status](https://img.shields.io/travis/okTurtles/group-income-simple/master.svg)](https://travis-ci.org/okTurtles/group-income-simple) [![Deps](https://david-dm.org/okTurtles/group-income-simple.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=dependencies) [![Dev Deps](https://david-dm.org/okTurtles/group-income-simple/dev-status.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=devDependencies)
 
+[![Bitcoin](https://img.shields.io/badge/donate-BTC-blue.svg)](https://okturtles.org/donate/) [![ETH](https://img.shields.io/badge/donate-ETH-blue.svg)](https://okturtles.org/donate/) [![Litecoin](https://img.shields.io/badge/donate-LTC-blue.svg)](https://okturtles.org/donate/) [![XMR](https://img.shields.io/badge/donate-XMR-blue.svg)](https://okturtles.org/donate/) [![Fiat](https://img.shields.io/badge/donate-%24%20%C2%A3%20%E2%82%AC%20%C2%A5-blue.svg)](https://okturtles.org/donate/)
```
Comment (taoeffect) [inline README.md]: Lets just have a single donate button here with `Donate` in the grey section and ❤️ in the blue section?

File: README.md [hunk 2]
```diff
@@ -75,6 +77,10 @@ For details, see: **[`Docker.md`](docs/Docker.md)**
 
 If you run into any errors [during the setup](docs/Getting-Started-frontend.md#how-do-i-get-set-up--just-run-the-site), try the suggestions in [`Troubleshooting.md`](docs/Troubleshooting.md).
 
+## Donating
+
+Donations to [support development on Group Income](https://groupincome.org/donate/) and the [other work of the okTurtles Foundation](https://okturtles.org/donate/) are greatly appreciated.
```
Comment (taoeffect) [inline README.md]: Just one link to the okTurtles Foundation donation page here is fine. Text can say something like:

> Donations to the okTurtles Foundation go to supporting the development of Group Income and related projects

**Review by taoeffect — CHANGES_REQUESTED** [review]
See comments!

File: README.md (hunk 1 above)
Comment (dotmacro) [reply README.md]: shields.io only supports colored text/emoji/logos on the lefthand side of the button. The ❤️ also ends up off-center:


One option is to use other emoji/text (but not spaces) to center the righthand side:


Alternatively, if we want a red ❤️, we can create a custom svg that resembles the other buttons.

Comment (dotmacro) [reply README.md]: From our chat: 
> Lets use this one:
>

**Comment by taoeffect** [comment]
This is almost ready to merge, only one issue: when I view [the preview of the file](https://github.com/okTurtles/group-income-simple/blob/9a9dd445683d0e589cd450ecba35d2e98b469210/README.md) in safari, this is what I see:

screen shot 2018-09-14 at 6 20 35 pm

There's a little square on the end of the button for some reason.

**Comment by dotmacro** [comment]
Good catch! I removed the utf-8 that wasn't being rendered. Button now renders properly in Safari, Firefox, and Chrome.

**Pull request merged**